routes/v1/stats.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import Stats from '../../controllers/stats';
+import Validation from '../../validators/articles';
+
+const router = express.Router();
+
+router.patch('/:slug', Validation.articleSlug, Stats.update);
```

Comment:
Since you are updating articles tag, It will be better you declare this route in the articles route.

Reply:
done